spec/models/category_spec.rb

```diff
@@ -1,5 +1,26 @@
 require 'spec_helper'
 
 describe Category do
-  pending "add some examples to (or delete) #{__FILE__}"
+
+  it {should have_many(:videos)}
+
+  # km01:out for shoulda
+  # it "a category can save itself" do
+  #   category = Category.new(name:"comedies")
+  #   category.save
+  #   expect(Category.first).to eq(category)
+  # end
+  #
```

Comment:
These should not be commented out. The shoulda test here only confirms that the relationship is created in the model.